tests_and_examples/run_query_tests.py

```diff
@@ -35,8 +35,7 @@ class SlicingDiceTester(object):
     """Test orchestration class."""
     def __init__(self, api_key, verbose=False, endpoint_test=True):
         # The Slicing Dice API client
-        self.client = SlicingDice(api_key)
-        self.endpoint_test = endpoint_test
+        self.client = SlicingDice(master_key=api_key, test=endpoint_test)
```

Comment:
rename 'endpoint_test' to
'uses_test_endpoint'

make clear this is a boolean variable